src/app/devices/device-detail/device-detail.component.html

```diff
@@ -1,34 +1,141 @@
-        <div fxLayoutAlign="center" style="margin-top:12px">
-            <button mat-flat-button (click)="navigateTo('audit-log')" color="primary" style="margin: 5px">See All Activity</button>
-        </div>
+                <mat-vertical-stepper #stepper>
+                    <mat-step completed="false" editable="false"
+                        *ngFor="let entry of auditLogData.records.slice().reverse()">
+                        <ng-template matStepLabel><span data-cy="auditLogEntry">{{entry.Event}}</span><br />{{
+                            entry.Time | amTimeAgo:true }} ago</ng-template>
+                    </mat-step>
+                </mat-vertical-stepper>
+                <div fxLayoutAlign="center" style="margin-top:12px">
+                    <button mat-flat-button (click)="navigateTo('audit-log')" color="primary" style="margin: 5px">See
+                        All Activity</button>
+                </div>
             </mat-tab>
             <mat-tab label="Event Log" class="tab">
                 <h3 class="mat-h3">Device Event Log ({{eventLogData.length}} entries) </h3>
                 <mat-vertical-stepper #stepper>
                     <mat-step completed="false" editable="false" *ngFor="let entry of eventLogData.slice(0, 10)">
-                        <ng-template matStepLabel><span data-cy="eventLogEntry">{{entry.Desc}}</span><br />{{ entry.Time | amTimeAgo:true }} ago</ng-template>
+                        <ng-template matStepLabel><span data-cy="eventLogEntry">{{entry.Desc}}</span><br />{{ entry.Time
+                            | amTimeAgo:true }} ago</ng-template>
                     </mat-step>
                 </mat-vertical-stepper>
                 <div fxLayoutAlign="center" style="margin-top:12px">
-                    <button mat-flat-button (click)="navigateTo('event-log')" color="primary" style="margin: 5px"> <span><div data-cy="eventSeeAllActivity">See All Activity</div></span></button>
+                    <button mat-flat-button (click)="navigateTo('event-log')" color="primary" style="margin: 5px">
+                        <span>
+                            <div data-cy="eventSeeAllActivity">See All Activity</div>
+                        </span></button>
                 </div>
             </mat-tab>
         </mat-tab-group>
     </mat-sidenav>
     <mat-sidenav-content>
+        <div fxLayout="row wrap">
+            <mat-card fxFlex="100">
+                <mat-card-header>
+                    <mat-card-title>
+                        Out-of-Band Power Actions
+                    </mat-card-title>
+                    <mat-card-subtitle>Actions occur at the hardware level. Use these
+                        with caution as they are akin to disconnecting the power cable from the device.</mat-card-subtitle>
+
+                </mat-card-header>
+                <mat-card-content>
+                    <!--
+                    <button *ngFor="let option of powerOptions" mat-flat-button>
```

Comment:
i was thinking we could keep these buttons data driven. Is there a particular reason why you didn't want to do that?